src/app/components/SubjectHeading/SubjectHeadingsTableBody.jsx

```diff
@@ -30,26 +30,18 @@ class SubjectHeadingsTableBody extends React.Component {
     this.backgroundColor = this.backgroundColor.bind(this);
   }
 
-  componentDidUpdate() {
-    window.tbody = this;
-    if (!this.state.subjectHeadings && this.props.subjectHeadings) {
```

Comment:
now we automatically set `subjectHeadings` in state. So, this conditional would never be true.